packages/redux/src/users/addresses/actions/factories/removeUserDefaultBillingAddressFactory.ts

```diff
@@ -0,0 +1,52 @@
+import * as actionTypes from '../../actionTypes.js';
+import {
+  type Config,
+  type DeleteUserDefaultBillingAddress,
+  toBlackoutError,
+  type User,
+  type UserAddress,
+} from '@farfetch/blackout-client';
+import type { Dispatch } from 'redux';
+import type { RemoveUserDefaultBillingAddressAction } from '../../types/index.js';
+
+/**
+ * Responsible for deleting the users default billing address.
+ *
+ * @param deleteUserDefaultBillingAddress - Delete default Billing address client.
+ *
+ * @returns Thunk factory.
+ */
+const removeUserDefaultBillingAddressFactory =
+  (deleteUserDefaultBillingAddress: DeleteUserDefaultBillingAddress) =>
+  (userId: User['id'], addressId: UserAddress['id'], config?: Config) =>
```

Comment:
Since the client does not need the addressId parameter, it should not be required here either. If you need it for the reducer to update the state, you will need to tackle that in another way (for example by looking inside the user's address currently on the store and finding the one that is currently set as the default billing address and update the flag to false.)